### DISCUSSION TITLE
Installing GSAP from private NPM using PNPM not working

### DISCUSSION THREAD

**Discussion by StevenStavrakis** [vercel/community#4278] [Help]
### Summary

Having followed all instructions for using private NPM packages, I'm still getting errors blocking successful builds.

Here is the error code:

``` WARN  GET https://registry.npmjs.org/@gsap/shockingly/-/shockingly-3.12.2.tgz error (ERR_PNPM_FETCH_404). Will retry in 10 seconds. 2 retries left.```

My NPM_RC env variable:
```
always-auth=true
registry=https://registry.npmjs.org
@gsap:registry=https://npm.greensock.com
//npm.greensock.com/:_authToken=MY_API_KEY_HERE
```

### Steps to Reproduce

1. Create an Astro project using PNPM
2. Add GSAP with private registry
3. Use .npmrc as a vercel env variable via `vc env add NPM_RC development < .npmrc`
4. Wait for deploy to fail

**Comment by StevenStavrakis** [comment]
Changing to NPM seems to fix the issue, but I don't consider this a resolution

**Comment by feledori** [comment]
stuck with the same error
